core/src/test/scala/za/co/absa/spline/harvester/dispatcher/ConsoleLineageDispatcherSpec.scala

```diff
@@ -33,31 +33,15 @@ class ConsoleLineageDispatcherSpec
 
   private val uuid1 = UUID.fromString("12345678-90ab-cdef-1234-567890abcdef")
 
-  it should "send lineage to the stdout (null planId)" in {
-    assertingStdOut(include("""["event",{"timestamp":999}]""")) {
-      new ConsoleLineageDispatcher(new BaseConfiguration {
-        addProperty("stream", "OUT")
-      }).send(ExecutionEvent(null, 999, None, None, None))
-    }
-  }
-
-  it should "send lineage to the stdout (non-null planId)" in {
+  it should "send lineage to the stdout" in {
     assertingStdOut(include("""["event",{"planId":"12345678-90ab-cdef-1234-567890abcdef","timestamp":999}]""")) {
       new ConsoleLineageDispatcher(new BaseConfiguration {
         addProperty("stream", "OUT")
       }).send(ExecutionEvent(uuid1, 999, None, None, None))
     }
   }
 
-  it should "send lineage to the stderr (null planId)" in {
-    assertingStdErr(include("""["event",{"timestamp":999}]""")) {
-      new ConsoleLineageDispatcher(new BaseConfiguration {
-        addProperty("stream", "ERR")
-      }).send(ExecutionEvent(null, 999, None, None, None))
-    }
-  }
-
-  it should "send lineage to the stderr (non-null planId)" in {
```

Comment:
Why is it no longer desirable to check that the json serialization result with the null fields has the expected form?

Reply:
That test was irrelevant anyway. The serialization logic should be tested on the serializer level. I'll add a test (thought I added one, but apparently I removed it during cleanup before committing)

Reply:
added. please check